base_location_nuts/security/ir.model.access.csv

```diff
@@ -1,3 +1,4 @@
 "id","name","model_id:id","group_id:id","perm_read","perm_write","perm_create","perm_unlink"
 "access_res_partner_nuts_user_all","res_partner_nuts group_partner_manager_all","model_res_partner_nuts","",1,0,0,0
 "access_res_partner_nuts_user","res_partner_nuts group_partner_manager","model_res_partner_nuts","base.group_partner_manager",1,1,1,1
+"access_nuts_import","nuts.import access","model_nuts_import","base.group_user",1,1,1,1
```

Comment:
I don't think all users access to this wizard. Can you restrict this a bit?

Reply:
for `group_partner_manager`?

Reply:
Done and update Readme